packages/quantic/force-app/main/default/lwc/quanticFacet/e2e/quanticFacet.e2e.ts

```diff
@@ -0,0 +1,209 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+import facetData from './data';
+
+const fixtures = {
+  search: testSearch as typeof testSearch,
+  insight: testInsight as typeof testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test;
+  if (useCase.value === useCaseEnum.search) {
+    test = fixtures[useCase.value] as typeof testSearch;
```

Comment:
if you add the `as typeof` above, I dont think you need it here then no?

Reply:
updated!